app/controllers/organisations_controller.rb

```diff
@@ -87,30 +87,54 @@ def update
     end
   end
 
-  def logs
+  def lettings_logs
     if current_user.support?
       set_session_filters(specific_org: true)
 
-      organisation_logs = LettingsLog.all.where(owning_organisation_id: @organisation.id)
-      unpaginated_filtered_logs = filtered_lettings_logs(filtered_collection(organisation_logs, search_term))
+      organisation_logs = LettingsLog.where(owning_organisation_id: @organisation.id)
+      unpaginated_filtered_logs = filtered_logs(filtered_collection(organisation_logs, search_term))
 
       respond_to do |format|
         format.html do
-          @pagy, @lettings_logs = pagy(unpaginated_filtered_logs)
+          @pagy, @logs = pagy(unpaginated_filtered_logs)
           @searched = search_term.presence
           @total_count = organisation_logs.size
           render "logs", layout: "application"
         end
 
         format.csv do
-          send_data byte_order_mark + unpaginated_filtered_logs.to_csv, filename: "logs-#{@organisation.name}-#{Time.zone.now}.csv"
+          send_data byte_order_mark + unpaginated_filtered_logs.to_csv, filename: "lettings-logs-#{@organisation.name}-#{Time.zone.now}.csv"
         end
       end
     else
       redirect_to(lettings_logs_path)
     end
   end
 
+  def sales_logs
```

Comment:
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/851 is about to be merged that changes this. Do you want to revise?